chunky/src/java/se/llbit/chunky/entity/Campfire.java

```diff
@@ -347,9 +349,14 @@ private static int getOrientationIndex(String facing) {
 
   @Override
   public Grid.EmitterPosition[] getEmitterPosition() {
+      if (block == null) {
+          Log.warn("Attempted to build emitter grid from unassociated campfire entity.");
+          return new Grid.EmitterPosition[0];
```

Comment:
This means that it won't work after loading an existing scene. Should we store the associated block position to fix this?

Reply:
No it should still work. The emitter grid file stores all the `EmitterPosition`s and can get all the information needed from there. This shouldn't ever be called once a scene is loaded.